pillarbox-core-business-cast/src/main/java/ch/srgssr/pillarbox/core/business/cast/SRGMediaItemConverter.kt

```diff
@@ -79,8 +80,9 @@ class SRGMediaItemConverter : MediaItemConverter {
                 ilHost?.let { host(it) }
             }
         } else {
+            val mediaId = if (mediaInfo.contentUrl == mediaInfo.contentId) MediaItem.DEFAULT_MEDIA_ID else mediaInfo.contentId
```

Comment:
So if the integrator explicitly sets the media id to the media URL, we also set the media id to `MediaItem.DEFAULT_MEDIA_ID`.
Is that ok?

Reply:
But if doesn't explicitly set a media id, it would have one. What to do....

Reply:
I think we can stay with the current solution: the `contentUrl` is always available in the uri, and the id is in the `mediaId` if it's different from `contentUrl`.
So resulting object may not be the same as the original one, but there is no loss of data.